src/helpers/airData.ts

```diff
@@ -0,0 +1,104 @@
+export const airData = [
```

Comment:
Thanks for making this so we don't have to continuously fetch the data while we style.